google/oauth2/credentials.py

```diff
@@ -126,6 +128,9 @@ def __init__(
             granted_scopes (Optional[Sequence[str]]): The scopes that were consented/granted by the user.
                 This could be different from the requested scopes and it could be empty if granted
                 and requested scopes were same.
+            trust_boundary (str): String representation of trust boundary meta.
```

Comment:
Is this a merge error?

Reply:
no, when I added the universe_domain doc string I noticed that the trust_boundary doc string was missing, so I just added it as well.